src/main/resources/hudson/plugins/sshslaves/SSHConnector/config.jelly

```diff
@@ -5,7 +5,9 @@
       <c:select checkMethod="post"/>
   </f:entry>
 
-  <f:dropdownDescriptorSelector title="${%Host Key Verification Strategy}" field="sshHostKeyVerificationStrategy" />
+  <f:entry>
```

Comment:
I think this is not really necessary when https://github.com/jenkinsci/jenkins/pull/6255 gets merged. There I faced the same issue and the solution was that the form needs the `class="jenkins-form"`

Reply:
worth waiting for this 👍

Reply:
So, revert this hunk and update to 2.343?